website/src/components/Showcase/Showcase.js

```diff
@@ -6,9 +6,9 @@ import styles from './Showcase.module.css';
 import _ from 'lodash';
 import { TailSpin } from 'react-loader-spinner';
 
-function Showcase() {
+const Showcase = () => {
   const [cards, setCards] = useState([]);
-  const [toRender, setToRender] = useState([]);
+  const [toRender, setToRender] = useState([<TailSpin />]);
 
   useEffect(() => {
     setCards(_makeShuffledCardList(cardList).map(_makeCard));
```

Comment:
I don't think that all those numerous changes were necessary. Consider this direction:

```suggestion
    const resultingCardList = shouldShuffle() ? _makeShuffledCardList(cardList) : cardList;
    setCards(cardList.map(_makeCard));
```

It is a bad idea to make `_makeShuffledCardList` so smart – despite a straightforward name, this function starts to decide whether it wants to shuffle or not. Besides, nothing implies that such a function would be DOM-dependent as its goal is just to shuffle the list.

Please go with a simpler version as suggested above. If there some things I didn't notice, let me know.

Reply:
I agree with Yarik, the logic of whether we should call `_makeShuffledCardList` or not should be extracted from `_makeShuffledCardList` to another method, otherwise the method's name is wrong.

Reply:
Very good advice here and thank you both. I see now. The logic to shuffle or not, should be handled outside of the _makeShuffledCardList function. I removed it and placed it inside useEffect and included your suggestion @noomorph 
Will push changes.